### PR TITLE
LPS-101643 Deprecate Liferay.Util.escapeCDATA

### PR DESCRIPTION
**Pull request by julien**
The goal of this change is to mark the `Liferay.Util.escapeCDATA` function as
deprecated. Based on the conversations we've had in
https://github.com/wincent/liferay-portal/pull/210, we're still keeping the function to avoid breaking changes in case it's being used externally.
I couldn't find any internal usages (I searched using `git grep
escapeCDATA`), but here is a list of commits that are related:
https://github.com/liferay/liferay-portal/commit/37ae29c447253b56d3f5bd6e966b19872a6bdab0
https://github.com/liferay/liferay-portal/commit/be13362f3bc39b3c6825a49cea525398a78d1238

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js [hunk 1]
```diff
@@ -392,6 +392,9 @@
 			Util.toggleDisabled(inputs, false);
 		},
 
+		/**
+		 * @deprecated As of Athanasius (7.3.x), with no replacement
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js:396]: "with no direct replacement"

**Comment by wincent** [comment]
ci:stop

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by wincent** [comment]
CI still seems unresponsive after 21 hours. Trying again.

**Comment by wincent** [comment]
ci:stop

**Comment by wincent** [comment]
ci:forward

**Comment by wincent** [comment]
"ci:test:relevant" failure is obviously nonsense (this is a comment-only change), it's saying "upstream comparison is not available" (and I see similar on [other unrelated pulls](https://github.com/izaera/liferay-portal/pull/139)). As this is a comment-only change, I am just going to forward manually because shipping trivial code should not be this difficult.

Link: https://github.com/brianchandotcom/liferay-portal/pull/87903

**Comment by wincent** [comment]
[Slack report](https://liferay.slack.com/archives/C19PWQ74J/p1587643558299500), for posterity.

**Pull request closed without merging**